tests/unit/amo/helpers.js

```diff
@@ -82,6 +83,7 @@ export const fakeTheme = Object.freeze({
   authors: [{
     name: 'Madonna',
```

Comment:
This is unrelated but MaDonna is so great we should respect her real name. The "D" here should be uppercase; [that's how she styles it](https://addons.mozilla.org/en-GB/firefox/user/MaDonna/themes).

Technically her username is MaDonna too. 😄